docs/design/latency_definitions/callback.md

```diff
@@ -0,0 +1,40 @@
+# Callback
+
+The callback latency is the time difference from callback start to callback end.
+
+$$
+l_{\rm{callback}} = t_{\rm{callback\ end}} - t_{\rm{callback\ start}}
+$$
+
+A simplified sequence diagram focusing only on the relevant data flow is shown below.
```

Comment:
I think that it is confusing the sequence diagram is designed to show data flow. 

```suggestion
The sequence diagram shows how CARET rclcpp pick up two events; callback_start and callback_end 
```

Reply:
addressed: [ea6f556](https://github.com/tier4/CARET_doc/pull/106/commits/ea6f556a84802265fdd6222bee05a63aa0620a68)